CHANGELOG.md

```diff
@@ -2,6 +2,34 @@
 
 The release log for BoTorch.
 
+## [0.9.5] -- Dec 8, 2023
+
+#### New features
+
+Hypervolume Knowledge Gradient (HVKG):
+* Add `qHypervolumeKnowledgeGradient`, which seeks to maximize the difference in hypervolume of the hypervolume-maximizing set of a fixed size after conditioning the unknown observation(s) that would be received if X were evaluated (#1950).
```

Comment:
I find this sentence a bit hard to follow but looks like this is also what is in the docstring. Probably ok to leave as is.